### PR TITLE
feat(cli): allow status app url to be configured

### PR DESCRIPTION
**Pull request by KnisterPeter**
Related to #371

### CODE REVIEW

**Comment by KnisterPeter** [comment]
@googlebot I signed it!

File: packages/cli/src/upload/upload.js [hunk 1]
```diff
@@ -145,13 +151,13 @@ async function postStatusToGitHub(options) {
     githubToken,
     githubAppToken,
     githubApiHost = DEFAULT_GITHUB_API_HOST,
+    githubAppUrl = GITHUB_APP_STATUS_CHECK_URL,
```
Comment (patrickhulce) [inline packages/cli/src/upload/upload.js:154]: I think you'll need to thread this option through in `runGithubStatusCheck` as well

File: packages/cli/src/upload/upload.js [hunk 2]
```diff
@@ -128,11 +128,17 @@ function buildCommand(yargs) {
       description: '[filesystem only] The pattern to use for naming Lighthouse reports.',
       default: '%%HOSTNAME%%-%%PATHNAME%%-%%DATETIME%%.report.%%EXTENSION%%',
     },
+    githubAppUrl: {
+      type: 'string',
+      default: GITHUB_APP_STATUS_CHECK_URL,
+      description:
+        'The url of the github app to post to for receiving status checks per PR. Defaults to the lighthouse hosted github app.',
```
Comment (patrickhulce) [inline packages/cli/src/upload/upload.js:135]: some minor tweaks

```suggestion
        'The URL of the GitHub app where PR status checks are POST'd. Defaults to the publicly hosted app. Most users will not need to change this.',
```

File: packages/cli/src/upload/upload.js (hunk 1 above)
Comment (KnisterPeter) [reply packages/cli/src/upload/upload.js:154]: Right

File: packages/cli/src/upload/upload.js (hunk 2 above)
Comment (KnisterPeter) [reply packages/cli/src/upload/upload.js]: Done

**Comment by KnisterPeter** [comment]
@patrickhulce Your requests are changed. You can re-review

File: packages/cli/src/upload/upload.js [hunk 3]
```diff
@@ -128,11 +128,17 @@ function buildCommand(yargs) {
       description: '[filesystem only] The pattern to use for naming Lighthouse reports.',
       default: '%%HOSTNAME%%-%%PATHNAME%%-%%DATETIME%%.report.%%EXTENSION%%',
     },
+    githubAppUrl: {
+      type: 'string',
+      default: GITHUB_APP_STATUS_CHECK_URL,
+      description:
+        "The URL of the GitHub app where PR status checks are POST'd. Defaults to the publicly hosted app. Most users will not need to change this.",
+    },
```
Comment (jflayhart) [inline packages/cli/src/upload/upload.js:136]: 🎉

**Pull request merged**